core/system/inc/mpu/vmpu_exports.h

```diff
@@ -88,18 +88,23 @@
 #define UVISOR_MIN_STACK_SIZE       1024
 #define UVISOR_MIN_STACK(x)         (((x)<UVISOR_MIN_STACK_SIZE)?UVISOR_MIN_STACK_SIZE:(x))
 
-#if defined(ARCH_MPU_ARMv7M) || (defined(YOTTA_CFG_UVISOR_PRESENT) && !defined(TARGET_LIKE_KINETIS))
+#if defined(ARCH_MPU_ARMv7M) || (YOTTA_CFG_UVISOR_PRESENT == 1 && !defined(TARGET_LIKE_KINETIS))
 #define UVISOR_REGION_ROUND_DOWN(x) ((x) & ~((1UL << UVISOR_REGION_BITS(x)) - 1))
 #define UVISOR_REGION_ROUND_UP(x)   (1UL << UVISOR_REGION_BITS(x))
 #define UVISOR_STACK_SIZE_ROUND(x)  UVISOR_REGION_ROUND_UP(x)
-#elif defined(ARCH_MPU_KINETIS) || (defined(YOTTA_CFG_UVISOR_PRESENT) && defined(TARGET_LIKE_KINETIS))
+#elif defined(ARCH_MPU_KINETIS) || (YOTTA_CFG_UVISOR_PRESENT == 1 && defined(TARGET_LIKE_KINETIS))
 #define UVISOR_REGION_ROUND_DOWN(x) ((x) & ~0x1FUL)
 #define UVISOR_REGION_ROUND_UP(x)   UVISOR_REGION_ROUND_DOWN((x) + 31UL)
 #define UVISOR_STACK_SIZE_ROUND(x)  UVISOR_REGION_ROUND_UP((x) + (UVISOR_STACK_BAND_SIZE * 2))
+#elif !defined(YOTTA_CFG_UVISOR_PRESENT) || YOTTA_CFG_UVISOR_PRESENT == 0
+#define UVISOR_REGION_ROUND_DOWN(x) (x)
+#define UVISOR_REGION_ROUND_UP(x)   (x)
+#define UVISOR_STACK_SIZE_ROUND(x)  (x)
 #else
 #error "Unknown MPU architecture. uvisor: Check your Makefile. uvisor-lib: Check if uVisor is supported"
 #endif /* defined(ARCH_MPU_ARMv7M) || (defined(YOTTA_CFG_UVISOR_PRESENT) && !defined(TARGET_LIKE_KINETIS)) or
-          defined(ARCH_MPU_KINETIS) || (defined(YOTTA_CFG_UVISOR_PRESENT) && defined(TARGET_LIKE_KINETIS)) */
+          defined(ARCH_MPU_KINETIS) || (defined(YOTTA_CFG_UVISOR_PRESENT) && defined(TARGET_LIKE_KINETIS)) or
+          !defined(YOTTA_CFG_UVISOR_PRESENT) || YOTTA_CFG_UVISOR_PRESENT == 0 */
```

Comment:
This comment doesn't appear to match the `#if`. I'd recommend deleting the comment. I would rather we not add these sorts of comments and not require them in our style guide, since they are a pain to maintain and any decent text editor can indicate which `#endif` goes with which `#if`.